docs/Advanced-Usage/Manager/Manager-Tasks.rst

```diff
@@ -101,9 +101,9 @@ running this command.
 This command allows you to share AGFIs that you have already built (that are
 listed in :ref:`config-hwdb`) with other users. It will take the
 named hardware configurations that you list in the ``[agfistoshare]`` section of
-``config_build.ini``, grab the respective AGFIs for each from
-``config_hwdb.ini``, and share them across all F1 regions with the users listed
-in the ``[sharewithaccounts]`` section of ``config_build.ini``. You can also specify ``public=public`` in ``[sharewithaccounts]`` to make the AGFIs public.
+``config_build.yaml``, grab the respective AGFIs for each from
+``config_hwdb.yaml``, and share them across all F1 regions with the users listed
+in the ``[sharewithaccounts]`` section of ``config_build.yaml``. You can also specify ``public=public`` in ``[sharewithaccounts]`` to make the AGFIs public.
```

Comment:
Missed some ini to YAML reformatting in the syntax here.
```suggestion
in the ``sharewithaccounts:`` section of ``config_build.yaml``. You can also specify ``public: public`` in ``sharewithaccounts:`` to make the AGFIs public.
```

Reply:
See other docs comments